blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubOrgFolderTest.java

```diff
@@ -1,457 +0,0 @@
-package io.jenkins.blueocean.blueocean_github_pipeline;
-
-import com.cloudbees.hudson.plugins.folder.AbstractFolderProperty;
-import com.cloudbees.plugins.credentials.CredentialsProvider;
-import com.cloudbees.plugins.credentials.CredentialsScope;
-import com.cloudbees.plugins.credentials.CredentialsStore;
-import com.cloudbees.plugins.credentials.common.StandardCredentials;
-import com.cloudbees.plugins.credentials.common.StandardUsernamePasswordCredentials;
-import com.cloudbees.plugins.credentials.domains.Domain;
-import com.cloudbees.plugins.credentials.domains.DomainSpecification;
-import com.cloudbees.plugins.credentials.impl.UsernamePasswordCredentialsImpl;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
-import com.mashape.unirest.http.exceptions.UnirestException;
-
-import hudson.model.ItemGroup;
-import hudson.model.TopLevelItem;
-import hudson.model.TopLevelItemDescriptor;
-import hudson.model.User;
-import io.jenkins.blueocean.credential.CredentialsUtils;
-import io.jenkins.blueocean.rest.factory.organization.OrganizationFactory;
-import io.jenkins.blueocean.rest.impl.pipeline.credential.BlueOceanCredentialsProvider;
-import io.jenkins.blueocean.rest.impl.pipeline.credential.BlueOceanDomainRequirement;
-import io.jenkins.blueocean.rest.impl.pipeline.credential.BlueOceanDomainSpecification;
-import io.jenkins.blueocean.rest.model.BlueOrganization;
-import io.jenkins.blueocean.service.embedded.OrganizationFactoryImpl;
-import io.jenkins.blueocean.service.embedded.rest.OrganizationImpl;
-import jenkins.branch.OrganizationFolder;
-import jenkins.model.Jenkins;
-import jenkins.model.ModifiableTopLevelItemGroup;
-
-import org.apache.commons.lang.StringUtils;
-import org.jenkinsci.plugins.github_branch_source.Connector;
-import org.junit.Assert;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.junit.runners.Parameterized;
-import org.junit.runners.Parameterized.Parameters;
-import org.jvnet.hudson.test.MockFolder;
-import org.jvnet.hudson.test.TestExtension;
-
-import java.io.IOException;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Map;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
-import static org.junit.Assert.assertTrue;
-
-
-/**
- * @author Vivek Pandey
- */
-@RunWith(Parameterized.class)
-public class GithubOrgFolderTest extends GithubMockBase {
```

Comment:
Do I read the diff correctly that these tests were just migrated over to `GithubPipelineCreateRequestTest`?